Dockerfile

```diff
@@ -1,30 +1,39 @@
 FROM alpine AS builder
 
+RUN mkdir -p /opt
+
 ARG IMAGE_ARCH=amd64
 
 ARG RCON_CLI_VERSION=1.4.4
 
 ADD https://github.com/itzg/rcon-cli/releases/download/${RCON_CLI_VERSION}/rcon-cli_${RCON_CLI_VERSION}_linux_${IMAGE_ARCH}.tar.gz /tmp/rcon-cli.tar.gz
 
-RUN mkdir -p /opt/rcon-cli && \
-    tar x -f /tmp/rcon-cli.tar.gz -C /opt/rcon-cli && \
-    rm /tmp/rcon-cli.tar.gz
+RUN tar x -f /tmp/rcon-cli.tar.gz -C /opt/ && \
+    chmod +x /opt/rcon-cli
```

Comment:
While you're in here could switch this over to my [easy-add](https://github.com/itzg/easy-add) I created recently; however, I don't have support for zip files yet to support the rclone install below...that would be pretty easy to implement if you're interested in switching these over.

Reply:
I can do that, although I personally prefer using ADD / COPY. The main reason is that it's hash will always be checked, even if the URL doesn't change. But I guess that when providing explicit versions, we can assume they won't be changed.